third-party/mosquitto/include/ifaddrs.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file
```

Comment:
It seems it's enough to add `include embox.net.lib.getifaddrs` into mods.conf or add depends to the module description (Mybuild)

Reply:
Let me check this

Reply:
The ifaddrs.h file was deleted.